app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,55 @@
     font-weight: normal;
   }
 }
+
+.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  li{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space